discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/Field.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * Field.
+ */
+public class Field extends GenericModel {
```

Comment:
I had to custom-write a type adapter otherwise Gson blows up when it tries to deserialize this JSON (specifically, type is not a string always, so the default JSON tokenizer throws `IllegalArgumentExcpetion`)